services/addons/tests/firmware_manager/test_download_blob.py

```diff
@@ -28,3 +28,21 @@ def test_download_gcp_blob(mock_storage_client, mock_logging):
     mock_logging.info.assert_called_with(
         "Downloaded storage object test.blob from bucket test-bucket to local file /home/test/."
     )
+
+
+@patch("addons.images.firmware_manager.download_blob.logging")
+def test_download_docker_blob(mock_logging):
+    # prepare
+    os.system = MagicMock()
+    blob_name = "test.blob"
+    destination_file_name = "/home/test/"
+
+    # run
+    download_blob.download_docker_blob(blob_name, destination_file_name)
+
+    # validate
+    os.system.assert_called_with(f"cp /mnt/blob_storage/{blob_name} {destination_file_name}")

```

Comment:
Do we need to validate file extension or type? What if providing .txt or some other files?

Reply:
That sounds like a good idea! File type validation checks have been added to `download_blob.py`